doc/en/components/scheduling/date-range-picker.md

```diff
@@ -0,0 +1,444 @@
+  today.getFullYear(),
+  today.getMonth(),
+  today.getDate() - 7 
+);
+const lastWeek: CustomDateRange[] = [
+  {
+    label: 'Previous 7 days',
+    dateRange: {
+      start: previousSeven,
+      end: today,
+    },
+  }
+]
+
+const dateRange = document.querySelector('igc-date-range-picker') as IgcDateRangePickerComponent;
+dateRange.customRanges = lastWeek;
+dateRange.usePredefinedRanges = true;
+```
+
+Now, when you click the newly created `"Previous 7 days"` chip in the calendar pop-up, the range will automatically be selected, starting from 7 days ago and ending today.
+
+`sample="/scheduling/date-range-picker/custom-ranges", height="500", alt="{Platform} Date Range Picker Custom Ranges Example"`
+
+### Disabled & Special dates
+
+You also have the ability to set disabled dates in the calendar to narrow the range of dates the user can choose from. To set the disabled dates, you can use the `DisabledDates` property.
+
+```ts
+let dateRange = document.querySelector('igc-date-range-picker') as IgcDateRangePickerComponent;
+
+const minDate = new Date(2025, 4, 1);
+const maxDate = new Date(2025, 4, 31);
+
+dateRange.disabledDates = [
+  {
+    type: DateRangeType.Between,
+    dateRange: [minDate, maxDate]
+  }
+] as DateRangeDescriptor[];
+```
+
+You can see more information about all the possibilities that the `DisabledDates` property offers here: [Disabled dates](./calendar.md#disabled-dates)
+
+You can also do the same if you want to set one or more special dates in the calendar; the only difference is that you need to use the `SpecialDates` property instead. [Special dates](./calendar.md#special-dates)
+
+### Forms
+
+The `DateRangePicker` component can also be used seamlessly with the HTML form element. The `Min`, `Max`, and `Required` properties act as form validators.
+
+`sample="/scheduling/date-range-picker/form", height="500", alt="{Platform} Date Range Picker Form Example"`
+
+## Additional configuration
+
+### Properties
+
+In addition to the properties we've already covered, the `DateRangePicker` component offers a variety of additional properties that allow you to further configure its behavior.
+
+<!-- WebComponents -->
+
+|Name|Type|Description|
+|--|--|--|
+| `Disabled` | boolean | Disables the component. |
+| `NonEditable` | boolean |	Disables typing in the input field(s). |
+| `Placeholder` | string | Placeholder text for the single input mode. |
+| `PlaceholderStart` | string | Placeholder text for the start date input (two inputs mode). |
+| `PlaceholderEnd` | string | Placeholder text for the end date input (two inputs mode). |
+| `Outlined` | boolean | Determines whether the input part will have outline appearance in the [Material theme](../themes/overview.md). |
+| `Prompt` | string | The prompt character used for unfilled parts of the input(s) mask. |
+| `ResourceStrings` | IgcCalendarResourceStrings & IgcDateRangePickerResourceStrings | Resource strings for localization of the date-range picker and the calendar. |
```

Comment:
```suggestion
| `ResourceStrings` | IgcDateRangePickerResourceStrings | Resource strings for localization of the date-range picker and the calendar. |
```
that was recently refactored and I have updated the spec - https://github.com/IgniteUI/igniteui-webcomponents/wiki/Date-Range-Picker#options